### DISCUSSION TITLE
Background styling

### DISCUSSION THREAD

**Discussion by peterhym21** [Blazored/Modal#485] [Q&A] [answered]
is there any easy way to set the background color in a simpel way

whenever i use the option Class it always breaks all the default styling even when i only set the background color, 
et even is broken in the sampels 

![image](https://user-images.githubusercontent.com/43846121/210356511-50f2bc4a-fb0d-4912-9514-8e8f929b620d.png)

**Comment by chrissainty** [comment, marked as answer]
I guess it depends on what you mean by easy 😉.

The way to change the backdrop colour is via CSS. You need to override the following class in your CSS file and specify the particular background colour you want to use. 

```css
﻿.bm-container {
    background-color: rgba(0,0,0,0.5);
}
```

For reference, the reason things break when you add a class via the `Class` parameter, is that it replaces the default classes with whatever class you provide. So unless you provide a class that styles all aspects of the modal you get results like those in your screenshot.

**Reply by peterhym21** [reply]
Thank you for clarifying what I thought

Does it use the default Modal form fx. bootstrap, 
or somting else 

and if so any where i can finde that css?

Thanks in advance fort the help :)

**Reply by peterhym21** [reply]
so there is no way to have 3 message modal 
![image](https://user-images.githubusercontent.com/43846121/210503532-521c1594-dc91-4fa1-9835-a041706f4a68.png)

where the background on each of them corresponds with red (error), light blue(Info) and orange /yellow (waring)

Because i can do this but that only does it for the car in the modal and not for the howl modal
![image](https://user-images.githubusercontent.com/43846121/210504011-e8519d3d-44eb-452c-bc3f-1e846ebe8d80.png)

![image](https://user-images.githubusercontent.com/43846121/210504247-9f95fe5c-0f82-4869-9352-0d7c91aa4f60.png)

**Reply by peterhym21** [reply]
No need for futher help Figuret it out

needed tom use `var options = new ModalOptions() { UseCustomLayout = true };`

and then make my ovn layout with my own css files
